modules/Reminders/Reminder.php

```diff
@@ -297,17 +291,30 @@ public static function addNotifications(jsAlerts $alert, $checkDecline = true)
         if ($popupReminders) {
             foreach ($popupReminders as $popupReminder) {
                 $relatedEvent = BeanFactory::getBean($popupReminder->related_event_module, $popupReminder->related_event_module_id);
+
+                if ($relatedEvent && isset($relatedEvent->date_start))
```

Comment:
Please keep your code to PSR-2 standard: http://www.php-fig.org/psr/psr-2/

If statements are a bit off as well as some other sections of the new code you've written, otherwise the fix works as expected. :+1: